decidim-elections/app/views/decidim/elections/censuses/_internal_users_form.html.erb

```diff
@@ -0,0 +1,52 @@
+<% if current_user %>
+  <% if @form.in_census? && @form.invalid? %>
+    <%= cell "decidim/announcement", title: t(".invalid"), body: t(".authorization_options_invalid"), callout_class: "alert" %>
+    <%= link_to t(".exit_button"), exit_path, class: "button button__secondary button__lg w-full mt-12" %>
+    <script>
+      document.addEventListener("DOMContentLoaded", function() {
+        const submit = document.querySelector("button[type='submit']");
+        if (submit) {
+          submit.disabled = true;
+        }
+      });
+    </script>
```

Comment:
Could we not have this? I am thinking where we render the form to actually add disabled property when the form is rendered?

Reply:
I'll have to render the submit inside the view, but it's ok. it gives more flexibility